scipy/signal/tests/test_signaltools.py

```diff
@@ -1268,6 +1270,7 @@ def test_dc(self, nx, ny, dtype):
         y = signal.resample(x, ny)
         assert_allclose(y, [1] * ny)
 
+    @pytest.mark.thread_unsafe
```

Comment:
This isn't failing for me locally. The mutable window is copied, so I think it should work? If you see a failure, can you show the traceback?

Reply:
This is another instance of the Cython error